src/docs/markdown/automatic-https.md

```diff
@@ -220,3 +222,83 @@ Before attempting any ACME transactions, Caddy will test the configured storage
 Caddy can obtain and manage wildcard certificates when it is configured to serve a site with a qualifying wildcard name. A site name qualifies for a wildcard if only its left-most domain label is a wildcard. For example, `*.example.com` qualifies, but these do not: `sub.*.example.com`, `foo*.example.com`, `*bar.example.com`, and `*.*.example.com`.
 
 To get a wildcard from Let's Encrypt, you simply need to enable the [DNS challenge](#dns-challenge) and use a wildcard domain in your config. We recommend using wildcards only when you have so many subdomains that you would encounter CA rate limits trying to obtain certificates for them all.
+
+
+
+## Examples
+
+Caddy implicitly uses the HTTPS port (default 443) for your [server addresses](/docs/conventions#network-addresses) that don't assign an explicit port (which would disable automatic HTTPS). 
+
+The global setting `auto_https` has two values:
+- `disable_redirects` adds an implicit HTTP port redirect.
+- `off` disables automatic HTTPS, default implicit port changes to HTTP for all server addresses.
```

Comment:
```suggestion
- `off` which disables automatic HTTPS altogether, including the HTTP->HTTPS redirect and automatic enabling of TLS for sites that meet the requirements.
```

Reply:
Not worth mentioning here the implicit default port being switched from HTTPS to HTTP? I guess that's still covered in the table directly below.

Reply:
Technically it doesn't do that.

```
{
        auto_https off
}

localhost

respond "Hello World"
```

Adapts to:

```json
{
  "apps": {
    "http": {
      "servers": {
        "srv0": {
          "listen": [
            ":443"
          ],

          "routes": [
            {
              "match": [
                {
                  "host": [
                    "localhost"
                  ]
                }
              ],
              "handle": [
                {
                  "handler": "subroute",
                  "routes": [
                    {
                      "handle": [
                        {
                          "body": "Hello World",
                          "handler": "static_response"
                        }
                      ]
                    }
                  ]
                }
              ],
              "terminal": true
            }
          ],
          "tls_connection_policies": [
            {}
          ],
          "automatic_https": {
            "disable": true
          }
        }
      }
    }
  }
}
```

It still only listens to `:443` but TLS will not be automatically managed (so connections will fail). So turning off `auto_https` in that situation only really makes sense if you also manually configure TLS (with your own cert/key for example).

Reply:
@francislavoie ah ok... thanks for clarifying that. I could have sworn I had tested it previously and had HTTP work implicitly, it seems I'm mistaken. Tried looking for mentions about this in the docs, but again can't seem to find that information I thought I had read of the default implicit port changing.

So the default port is always implicitly `443`? (Unless some other explicit configuration like a protocol scheme is provided)

I should probably clarify that in the docs if its not already.